bench/gl-stats.html

```diff
@@ -0,0 +1,78 @@
+<!doctype html>
+<html>
+<head>
+<meta charset="utf-8">
+<script src="../dist/mapbox-gl.js"></script>
+<style>.mapboxgl-canary { background-color: salmon; }</style>
+</head>
+<body style="margin: 0; padding: 0"><div id="map" style="width: 500px; height: 500px"></div></body>
+<script>
+mapboxgl.accessToken = 'pk.eyJ1IjoibW91cm5lciIsImEiOiJWWnRiWG1VIn0.j6eccFHpE3Q04XPLI7JxbA';
```

Comment:
Would be good to use an environment variable instead of hardcoding the access token here. I'm not sure what the best way to do that with our CI setup is.